---
layout: default
title: rCRS Eager
---

## IPN run, rCRS (momia1,3,5,6)

```
NXF_VER=22.10.6 nextflow run nf-core/eager -profile docker --input 'momias.tsv'  --fasta /references/mito/rCRS.fasta \
--max_memory 60GB --max_cpus 16 --max_time 288.h -config nextflow_custom.config --run_genotyping true --save_reference \
--genotyping_tool 'ug' --run_mtnucratio true --run_nuclear_contamination true --mapper bwamem --run_bam_filtering \
--bam_unmapped_type 'discard' --run_mapdamage_rescaling --run_vcf2genome  --vcf2genome_header chrM \
--bwa_index /references/mito/ --fasta_index /references/mito/rCRS.fasta.fai   -resume
```

### multiqc

-   [multiqc_report.html](ipn/multiqc/multiqc_report.html)

#### multiqc_data

-   [mqc_ar_length_count_plot_All.txt](ipn/multiqc/multiqc_data/mqc_ar_length_count_plot_All.txt)
-   [mqc_ar_length_count_plot_Collapsed.txt](ipn/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed.txt)
-   [mqc_ar_length_count_plot_Collapsed_Truncated.txt](ipn/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed_Truncated.txt)
-   [mqc_ar_length_count_plot_Discarded.txt](ipn/multiqc/multiqc_data/mqc_ar_length_count_plot_Discarded.txt)
-   [mqc_ar_length_count_plot_Mate1.txt](ipn/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate1.txt)
-   [mqc_ar_length_count_plot_Mate2.txt](ipn/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate2.txt)
-   [mqc_ar_length_count_plot_Singleton.txt](ipn/multiqc/multiqc_data/mqc_ar_length_count_plot_Singleton.txt)
-   [mqc_ar_retained_plot_1.txt](ipn/multiqc/multiqc_data/mqc_ar_retained_plot_1.txt)
-   [mqc_bcftools-stats-subtypes_1.txt](ipn/multiqc/multiqc_data/mqc_bcftools-stats-subtypes_1.txt)
-   [mqc_bcftools_stats_depth_1.txt](ipn/multiqc/multiqc_data/mqc_bcftools_stats_depth_1.txt)
-   [mqc_bcftools_stats_indel-lengths_1.txt](ipn/multiqc/multiqc_data/mqc_bcftools_stats_indel-lengths_1.txt)
-   [mqc_bcftools_stats_vqc_Count_Indels.txt](ipn/multiqc/multiqc_data/mqc_bcftools_stats_vqc_Count_Indels.txt)
-   [mqc_bcftools_stats_vqc_Count_SNP.txt](ipn/multiqc/multiqc_data/mqc_bcftools_stats_vqc_Count_SNP.txt)
-   [mqc_bcftools_stats_vqc_Count_Transitions.txt](ipn/multiqc/multiqc_data/mqc_bcftools_stats_vqc_Count_Transitions.txt)
-   [mqc_bcftools_stats_vqc_Count_Transversions.txt](ipn/multiqc/multiqc_data/mqc_bcftools_stats_vqc_Count_Transversions.txt)
-   [mqc_fastqc_adapter_content_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_adapter_content_plot_1.txt)
-   [mqc_fastqc_overrepresented_sequences_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_overrepresented_sequences_plot_1.txt)
-   [mqc_fastqc_per_base_n_content_plot-2_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot-2_1.txt)
-   [mqc_fastqc_per_base_n_content_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot-2_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot-2_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot_1.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Counts.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot_1.txt)
-   [mqc_fastqc_sequence_counts_plot-2_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot-2_1.txt)
-   [mqc_fastqc_sequence_counts_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot-2_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot-2_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot-2_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot-2_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot_1.txt)
-   [mqc_fiveprime_misinc_plot_1.txt](ipn/multiqc/multiqc_data/mqc_fiveprime_misinc_plot_1.txt)
-   [mqc_length-distribution-Forward_1.txt](ipn/multiqc/multiqc_data/mqc_length-distribution-Forward_1.txt)
-   [mqc_length-distribution-Reverse_1.txt](ipn/multiqc/multiqc_data/mqc_length-distribution-Reverse_1.txt)
-   [mqc_picard_deduplication_1.txt](ipn/multiqc/multiqc_data/mqc_picard_deduplication_1.txt)
-   [mqc_preseq_plot_1.txt](ipn/multiqc/multiqc_data/mqc_preseq_plot_1.txt)
-   [mqc_qualimap_coverage_histogram_1.txt](ipn/multiqc/multiqc_data/mqc_qualimap_coverage_histogram_1.txt)
-   [mqc_qualimap_gc_content_1.txt](ipn/multiqc/multiqc_data/mqc_qualimap_gc_content_1.txt)
-   [mqc_qualimap_genome_fraction_1.txt](ipn/multiqc/multiqc_data/mqc_qualimap_genome_fraction_1.txt)
-   [mqc_threeprime_misinc_plot_1.txt](ipn/multiqc/multiqc_data/mqc_threeprime_misinc_plot_1.txt)
-   [multiqc.log](ipn/multiqc/multiqc_data/multiqc.log)
-   [multiqc_adapter_removal.txt](ipn/multiqc/multiqc_data/multiqc_adapter_removal.txt)
-   [multiqc_bcftools_stats.txt](ipn/multiqc/multiqc_data/multiqc_bcftools_stats.txt)
-   [multiqc_citations.txt](ipn/multiqc/multiqc_data/multiqc_citations.txt)
-   [multiqc_damageprofiler_metrics.txt](ipn/multiqc/multiqc_data/multiqc_damageprofiler_metrics.txt)
-   [multiqc_data.json](ipn/multiqc/multiqc_data/multiqc_data.json)
-   [multiqc_fastqc.txt](ipn/multiqc/multiqc_data/multiqc_fastqc.txt)
-   [multiqc_fastqc_1.txt](ipn/multiqc/multiqc_data/multiqc_fastqc_1.txt)
-   [multiqc_general_stats.txt](ipn/multiqc/multiqc_data/multiqc_general_stats.txt)
-   [multiqc_mtnucratio.txt](ipn/multiqc/multiqc_data/multiqc_mtnucratio.txt)
-   [multiqc_picard_dups.txt](ipn/multiqc/multiqc_data/multiqc_picard_dups.txt)
-   [multiqc_qualimap_bamqc_genome_results.txt](ipn/multiqc/multiqc_data/multiqc_qualimap_bamqc_genome_results.txt)
-   [multiqc_samtools_flagstat.txt](ipn/multiqc/multiqc_data/multiqc_samtools_flagstat.txt)
-   [multiqc_samtools_flagstat_1.txt](ipn/multiqc/multiqc_data/multiqc_samtools_flagstat_1.txt)
-   [multiqc_software_versions.txt](ipn/multiqc/multiqc_data/multiqc_software_versions.txt)
-   [multiqc_sources.txt](ipn/multiqc/multiqc_data/multiqc_sources.txt)
-   [picard_histogram.txt](ipn/multiqc/multiqc_data/picard_histogram.txt)
-   [picard_histogram_1.txt](ipn/multiqc/multiqc_data/picard_histogram_1.txt)
-   [picard_histogram_2.txt](ipn/multiqc/multiqc_data/picard_histogram_2.txt)
-   [preseq.txt](ipn/multiqc/multiqc_data/preseq.txt)

### consensus_sequence

-   [m0mia6.fasta.gz](ipn/consensus_sequence/m0mia6.fasta.gz)
-   [m0mia6.fasta_refmod.fasta.gz](ipn/consensus_sequence/m0mia6.fasta_refmod.fasta.gz)
-   [m0mia6.fasta_uncertainty.fasta.gz](ipn/consensus_sequence/m0mia6.fasta_uncertainty.fasta.gz)
-   [momia1.fasta.gz](ipn/consensus_sequence/momia1.fasta.gz)
-   [momia1.fasta_refmod.fasta.gz](ipn/consensus_sequence/momia1.fasta_refmod.fasta.gz)
-   [momia1.fasta_uncertainty.fasta.gz](ipn/consensus_sequence/momia1.fasta_uncertainty.fasta.gz)
-   [momia3.fasta.gz](ipn/consensus_sequence/momia3.fasta.gz)
-   [momia3.fasta_refmod.fasta.gz](ipn/consensus_sequence/momia3.fasta_refmod.fasta.gz)
-   [momia3.fasta_uncertainty.fasta.gz](ipn/consensus_sequence/momia3.fasta_uncertainty.fasta.gz)
-   [momia5.fasta.gz](ipn/consensus_sequence/momia5.fasta.gz)
-   [momia5.fasta_refmod.fasta.gz](ipn/consensus_sequence/momia5.fasta_refmod.fasta.gz)
-   [momia5.fasta_uncertainty.fasta.gz](ipn/consensus_sequence/momia5.fasta_uncertainty.fasta.gz)

## SRA (A234) run, rCRS (ancient0002-0004)

```
NXF_VER=22.10.6 nextflow run nf-core/eager -profile docker --input 'momias.tsv'  --fasta /references/mito/rCRS.fasta \
--max_memory 60GB --max_cpus 16 --max_time 288.h -config nextflow_custom.config --run_genotyping true --save_reference \
--genotyping_tool 'ug' --run_mtnucratio true --run_nuclear_contamination true --mapper bwamem --run_bam_filtering \
--bam_unmapped_type 'discard' --run_mapdamage_rescaling --run_vcf2genome  --vcf2genome_header chrM \
--bwa_index /references/mito/ --fasta_index /references/mito/rCRS.fasta.fai   -resume
```

### Full set of reads (not subsetted for read length), rCRS

#### multiqc

-   [multiqc_report.html](sra/full/multiqc/multiqc_report.html)

##### multiqc_data

-   [mqc_ar_length_count_plot_All.txt](sra/full/multiqc/multiqc_data/mqc_ar_length_count_plot_All.txt)
-   [mqc_ar_length_count_plot_Collapsed.txt](sra/full/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed.txt)
-   [mqc_ar_length_count_plot_Collapsed_Truncated.txt](sra/full/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed_Truncated.txt)
-   [mqc_ar_length_count_plot_Discarded.txt](sra/full/multiqc/multiqc_data/mqc_ar_length_count_plot_Discarded.txt)
-   [mqc_ar_length_count_plot_Mate1.txt](sra/full/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate1.txt)
-   [mqc_ar_length_count_plot_Mate2.txt](sra/full/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate2.txt)
-   [mqc_ar_length_count_plot_Singleton.txt](sra/full/multiqc/multiqc_data/mqc_ar_length_count_plot_Singleton.txt)
-   [mqc_ar_retained_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_ar_retained_plot_1.txt)
-   [mqc_bcftools-stats-subtypes_1.txt](sra/full/multiqc/multiqc_data/mqc_bcftools-stats-subtypes_1.txt)
-   [mqc_bcftools_stats_depth_1.txt](sra/full/multiqc/multiqc_data/mqc_bcftools_stats_depth_1.txt)
-   [mqc_bcftools_stats_indel-lengths_1.txt](sra/full/multiqc/multiqc_data/mqc_bcftools_stats_indel-lengths_1.txt)
-   [mqc_bcftools_stats_vqc_Count_Indels.txt](sra/full/multiqc/multiqc_data/mqc_bcftools_stats_vqc_Count_Indels.txt)
-   [mqc_bcftools_stats_vqc_Count_SNP.txt](sra/full/multiqc/multiqc_data/mqc_bcftools_stats_vqc_Count_SNP.txt)
-   [mqc_bcftools_stats_vqc_Count_Transitions.txt](sra/full/multiqc/multiqc_data/mqc_bcftools_stats_vqc_Count_Transitions.txt)
-   [mqc_bcftools_stats_vqc_Count_Transversions.txt](sra/full/multiqc/multiqc_data/mqc_bcftools_stats_vqc_Count_Transversions.txt)
-   [mqc_fastqc_adapter_content_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_adapter_content_plot_1.txt)
-   [mqc_fastqc_per_base_n_content_plot-2_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot-2_1.txt)
-   [mqc_fastqc_per_base_n_content_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot-2_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot-2_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot_1.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Counts.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot_1.txt)
-   [mqc_fastqc_sequence_counts_plot-2_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot-2_1.txt)
-   [mqc_fastqc_sequence_counts_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot-2_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot-2_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot_1.txt)
-   [mqc_fiveprime_misinc_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_fiveprime_misinc_plot_1.txt)
-   [mqc_length-distribution-Forward_1.txt](sra/full/multiqc/multiqc_data/mqc_length-distribution-Forward_1.txt)
-   [mqc_length-distribution-Reverse_1.txt](sra/full/multiqc/multiqc_data/mqc_length-distribution-Reverse_1.txt)
-   [mqc_picard_deduplication_1.txt](sra/full/multiqc/multiqc_data/mqc_picard_deduplication_1.txt)
-   [mqc_preseq_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_preseq_plot_1.txt)
-   [mqc_qualimap_coverage_histogram_1.txt](sra/full/multiqc/multiqc_data/mqc_qualimap_coverage_histogram_1.txt)
-   [mqc_qualimap_gc_content_1.txt](sra/full/multiqc/multiqc_data/mqc_qualimap_gc_content_1.txt)
-   [mqc_qualimap_genome_fraction_1.txt](sra/full/multiqc/multiqc_data/mqc_qualimap_genome_fraction_1.txt)
-   [mqc_threeprime_misinc_plot_1.txt](sra/full/multiqc/multiqc_data/mqc_threeprime_misinc_plot_1.txt)
-   [multiqc.log](sra/full/multiqc/multiqc_data/multiqc.log)
-   [multiqc_adapter_removal.txt](sra/full/multiqc/multiqc_data/multiqc_adapter_removal.txt)
-   [multiqc_bcftools_stats.txt](sra/full/multiqc/multiqc_data/multiqc_bcftools_stats.txt)
-   [multiqc_citations.txt](sra/full/multiqc/multiqc_data/multiqc_citations.txt)
-   [multiqc_damageprofiler_metrics.txt](sra/full/multiqc/multiqc_data/multiqc_damageprofiler_metrics.txt)
-   [multiqc_data.json](sra/full/multiqc/multiqc_data/multiqc_data.json)
-   [multiqc_fastqc.txt](sra/full/multiqc/multiqc_data/multiqc_fastqc.txt)
-   [multiqc_fastqc_1.txt](sra/full/multiqc/multiqc_data/multiqc_fastqc_1.txt)
-   [multiqc_general_stats.txt](sra/full/multiqc/multiqc_data/multiqc_general_stats.txt)
-   [multiqc_mtnucratio.txt](sra/full/multiqc/multiqc_data/multiqc_mtnucratio.txt)
-   [multiqc_picard_dups.txt](sra/full/multiqc/multiqc_data/multiqc_picard_dups.txt)
-   [multiqc_qualimap_bamqc_genome_results.txt](sra/full/multiqc/multiqc_data/multiqc_qualimap_bamqc_genome_results.txt)
-   [multiqc_samtools_flagstat.txt](sra/full/multiqc/multiqc_data/multiqc_samtools_flagstat.txt)
-   [multiqc_samtools_flagstat_1.txt](sra/full/multiqc/multiqc_data/multiqc_samtools_flagstat_1.txt)
-   [multiqc_software_versions.txt](sra/full/multiqc/multiqc_data/multiqc_software_versions.txt)
-   [multiqc_sources.txt](sra/full/multiqc/multiqc_data/multiqc_sources.txt)
-   [picard_histogram.txt](sra/full/multiqc/multiqc_data/picard_histogram.txt)
-   [picard_histogram_1.txt](sra/full/multiqc/multiqc_data/picard_histogram_1.txt)
-   [picard_histogram_2.txt](sra/full/multiqc/multiqc_data/picard_histogram_2.txt)
-   [preseq.txt](sra/full/multiqc/multiqc_data/preseq.txt)

#### consensus_sequence

-   [ancient0002.fasta.gz](sra/full/consensus_sequence/ancient0002.fasta.gz)
-   [ancient0002.fasta_refmod.fasta.gz](sra/full/consensus_sequence/ancient0002.fasta_refmod.fasta.gz)
-   [ancient0002.fasta_uncertainty.fasta.gz](sra/full/consensus_sequence/ancient0002.fasta_uncertainty.fasta.gz)
-   [ancient0003.fasta.gz](sra/full/consensus_sequence/ancient0003.fasta.gz)
-   [ancient0003.fasta_refmod.fasta.gz](sra/full/consensus_sequence/ancient0003.fasta_refmod.fasta.gz)
-   [ancient0003.fasta_uncertainty.fasta.gz](sra/full/consensus_sequence/ancient0003.fasta_uncertainty.fasta.gz)
-   [ancient0004.fasta.gz](sra/full/consensus_sequence/ancient0004.fasta.gz)
-   [ancient0004.fasta_refmod.fasta.gz](sra/full/consensus_sequence/ancient0004.fasta_refmod.fasta.gz)
-   [ancient0004.fasta_uncertainty.fasta.gz](sra/full/consensus_sequence/ancient0004.fasta_uncertainty.fasta.gz)

### SRA, reads < lt105bp, rCRS

#### multiqc

-   [multiqc_report.html](sra/lt105bp/multiqc/multiqc_report.html)

##### multiqc_data

-   [mqc_ar_length_count_plot_All.txt](sra/lt105bp/multiqc/multiqc_data/mqc_ar_length_count_plot_All.txt)
-   [mqc_ar_length_count_plot_Collapsed.txt](sra/lt105bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed.txt)
-   [mqc_ar_length_count_plot_Collapsed_Truncated.txt](sra/lt105bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed_Truncated.txt)
-   [mqc_ar_length_count_plot_Discarded.txt](sra/lt105bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Discarded.txt)
-   [mqc_ar_length_count_plot_Mate1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate1.txt)
-   [mqc_ar_length_count_plot_Mate2.txt](sra/lt105bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate2.txt)
-   [mqc_ar_length_count_plot_Singleton.txt](sra/lt105bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Singleton.txt)
-   [mqc_ar_retained_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_ar_retained_plot_1.txt)
-   [mqc_bcftools_stats_indel-lengths_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_bcftools_stats_indel-lengths_1.txt)
-   [mqc_fastqc_overrepresented_sequences_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_overrepresented_sequences_plot_1.txt)
-   [mqc_fastqc_per_base_n_content_plot-2_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot-2_1.txt)
-   [mqc_fastqc_per_base_n_content_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot-2_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot-2_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot_1.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Counts.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot_1.txt)
-   [mqc_fastqc_sequence_counts_plot-2_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot-2_1.txt)
-   [mqc_fastqc_sequence_counts_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot-2_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot-2_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot-2_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot-2_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot_1.txt)
-   [mqc_fiveprime_misinc_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_fiveprime_misinc_plot_1.txt)
-   [mqc_length-distribution-Forward_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_length-distribution-Forward_1.txt)
-   [mqc_length-distribution-Reverse_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_length-distribution-Reverse_1.txt)
-   [mqc_picard_deduplication_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_picard_deduplication_1.txt)
-   [mqc_preseq_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_preseq_plot_1.txt)
-   [mqc_qualimap_coverage_histogram_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_qualimap_coverage_histogram_1.txt)
-   [mqc_qualimap_gc_content_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_qualimap_gc_content_1.txt)
-   [mqc_qualimap_genome_fraction_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_qualimap_genome_fraction_1.txt)
-   [mqc_threeprime_misinc_plot_1.txt](sra/lt105bp/multiqc/multiqc_data/mqc_threeprime_misinc_plot_1.txt)
-   [multiqc.log](sra/lt105bp/multiqc/multiqc_data/multiqc.log)
-   [multiqc_adapter_removal.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_adapter_removal.txt)
-   [multiqc_bcftools_stats.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_bcftools_stats.txt)
-   [multiqc_citations.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_citations.txt)
-   [multiqc_damageprofiler_metrics.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_damageprofiler_metrics.txt)
-   [multiqc_data.json](sra/lt105bp/multiqc/multiqc_data/multiqc_data.json)
-   [multiqc_fastqc.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_fastqc.txt)
-   [multiqc_fastqc_1.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_fastqc_1.txt)
-   [multiqc_general_stats.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_general_stats.txt)
-   [multiqc_mtnucratio.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_mtnucratio.txt)
-   [multiqc_picard_dups.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_picard_dups.txt)
-   [multiqc_qualimap_bamqc_genome_results.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_qualimap_bamqc_genome_results.txt)
-   [multiqc_samtools_flagstat.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_samtools_flagstat.txt)
-   [multiqc_samtools_flagstat_1.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_samtools_flagstat_1.txt)
-   [multiqc_software_versions.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_software_versions.txt)
-   [multiqc_sources.txt](sra/lt105bp/multiqc/multiqc_data/multiqc_sources.txt)
-   [picard_histogram.txt](sra/lt105bp/multiqc/multiqc_data/picard_histogram.txt)
-   [picard_histogram_1.txt](sra/lt105bp/multiqc/multiqc_data/picard_histogram_1.txt)
-   [picard_histogram_2.txt](sra/lt105bp/multiqc/multiqc_data/picard_histogram_2.txt)
-   [preseq.txt](sra/lt105bp/multiqc/multiqc_data/preseq.txt)

#### consensus_sequence

-   [ancient0002_lt105bp.fasta.gz](sra/lt105bp/consensus_sequence/ancient0002_lt105bp.fasta.gz)
-   [ancient0002_lt105bp.fasta_refmod.fasta.gz](sra/lt105bp/consensus_sequence/ancient0002_lt105bp.fasta_refmod.fasta.gz)
-   [ancient0002_lt105bp.fasta_uncertainty.fasta.gz](sra/lt105bp/consensus_sequence/ancient0002_lt105bp.fasta_uncertainty.fasta.gz)
-   [ancient0003_lt105bp.fasta.gz](sra/lt105bp/consensus_sequence/ancient0003_lt105bp.fasta.gz)
-   [ancient0003_lt105bp.fasta_refmod.fasta.gz](sra/lt105bp/consensus_sequence/ancient0003_lt105bp.fasta_refmod.fasta.gz)
-   [ancient0003_lt105bp.fasta_uncertainty.fasta.gz](sra/lt105bp/consensus_sequence/ancient0003_lt105bp.fasta_uncertainty.fasta.gz)
-   [ancient0004_lt105bp.fasta.gz](sra/lt105bp/consensus_sequence/ancient0004_lt105bp.fasta.gz)
-   [ancient0004_lt105bp.fasta_refmod.fasta.gz](sra/lt105bp/consensus_sequence/ancient0004_lt105bp.fasta_refmod.fasta.gz)
-   [ancient0004_lt105bp.fasta_uncertainty.fasta.gz](sra/lt105bp/consensus_sequence/ancient0004_lt105bp.fasta_uncertainty.fasta.gz)

### SRA, reads < lt120bp, rCRS

#### multiqc

-   [multiqc_report.html](sra/lt120bp/multiqc/multiqc_report.html)

##### multiqc_data

-   [mqc_ar_length_count_plot_All.txt](sra/lt120bp/multiqc/multiqc_data/mqc_ar_length_count_plot_All.txt)
-   [mqc_ar_length_count_plot_Collapsed.txt](sra/lt120bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed.txt)
-   [mqc_ar_length_count_plot_Collapsed_Truncated.txt](sra/lt120bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed_Truncated.txt)
-   [mqc_ar_length_count_plot_Discarded.txt](sra/lt120bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Discarded.txt)
-   [mqc_ar_length_count_plot_Mate1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate1.txt)
-   [mqc_ar_length_count_plot_Mate2.txt](sra/lt120bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate2.txt)
-   [mqc_ar_length_count_plot_Singleton.txt](sra/lt120bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Singleton.txt)
-   [mqc_ar_retained_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_ar_retained_plot_1.txt)
-   [mqc_bcftools_stats_indel-lengths_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_bcftools_stats_indel-lengths_1.txt)
-   [mqc_fastqc_overrepresented_sequences_plot-2_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_overrepresented_sequences_plot-2_1.txt)
-   [mqc_fastqc_overrepresented_sequences_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_overrepresented_sequences_plot_1.txt)
-   [mqc_fastqc_per_base_n_content_plot-2_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot-2_1.txt)
-   [mqc_fastqc_per_base_n_content_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot-2_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot-2_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot_1.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Counts.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot_1.txt)
-   [mqc_fastqc_sequence_counts_plot-2_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot-2_1.txt)
-   [mqc_fastqc_sequence_counts_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot-2_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot-2_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot-2_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot-2_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot_1.txt)
-   [mqc_fiveprime_misinc_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_fiveprime_misinc_plot_1.txt)
-   [mqc_length-distribution-Forward_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_length-distribution-Forward_1.txt)
-   [mqc_length-distribution-Reverse_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_length-distribution-Reverse_1.txt)
-   [mqc_picard_deduplication_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_picard_deduplication_1.txt)
-   [mqc_preseq_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_preseq_plot_1.txt)
-   [mqc_qualimap_coverage_histogram_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_qualimap_coverage_histogram_1.txt)
-   [mqc_qualimap_gc_content_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_qualimap_gc_content_1.txt)
-   [mqc_qualimap_genome_fraction_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_qualimap_genome_fraction_1.txt)
-   [mqc_threeprime_misinc_plot_1.txt](sra/lt120bp/multiqc/multiqc_data/mqc_threeprime_misinc_plot_1.txt)
-   [multiqc.log](sra/lt120bp/multiqc/multiqc_data/multiqc.log)
-   [multiqc_adapter_removal.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_adapter_removal.txt)
-   [multiqc_bcftools_stats.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_bcftools_stats.txt)
-   [multiqc_citations.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_citations.txt)
-   [multiqc_damageprofiler_metrics.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_damageprofiler_metrics.txt)
-   [multiqc_data.json](sra/lt120bp/multiqc/multiqc_data/multiqc_data.json)
-   [multiqc_fastqc.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_fastqc.txt)
-   [multiqc_fastqc_1.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_fastqc_1.txt)
-   [multiqc_general_stats.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_general_stats.txt)
-   [multiqc_mtnucratio.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_mtnucratio.txt)
-   [multiqc_picard_dups.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_picard_dups.txt)
-   [multiqc_qualimap_bamqc_genome_results.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_qualimap_bamqc_genome_results.txt)
-   [multiqc_samtools_flagstat.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_samtools_flagstat.txt)
-   [multiqc_samtools_flagstat_1.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_samtools_flagstat_1.txt)
-   [multiqc_software_versions.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_software_versions.txt)
-   [multiqc_sources.txt](sra/lt120bp/multiqc/multiqc_data/multiqc_sources.txt)
-   [picard_histogram.txt](sra/lt120bp/multiqc/multiqc_data/picard_histogram.txt)
-   [picard_histogram_1.txt](sra/lt120bp/multiqc/multiqc_data/picard_histogram_1.txt)
-   [picard_histogram_2.txt](sra/lt120bp/multiqc/multiqc_data/picard_histogram_2.txt)
-   [preseq.txt](sra/lt120bp/multiqc/multiqc_data/preseq.txt)

#### consensus_sequence

-   [ancient0002_lt120bp.fasta.gz](sra/lt120bp/consensus_sequence/ancient0002_lt120bp.fasta.gz)
-   [ancient0002_lt120bp.fasta_refmod.fasta.gz](sra/lt120bp/consensus_sequence/ancient0002_lt120bp.fasta_refmod.fasta.gz)
-   [ancient0002_lt120bp.fasta_uncertainty.fasta.gz](sra/lt120bp/consensus_sequence/ancient0002_lt120bp.fasta_uncertainty.fasta.gz)
-   [ancient0003_lt120bp.fasta.gz](sra/lt120bp/consensus_sequence/ancient0003_lt120bp.fasta.gz)
-   [ancient0003_lt120bp.fasta_refmod.fasta.gz](sra/lt120bp/consensus_sequence/ancient0003_lt120bp.fasta_refmod.fasta.gz)
-   [ancient0003_lt120bp.fasta_uncertainty.fasta.gz](sra/lt120bp/consensus_sequence/ancient0003_lt120bp.fasta_uncertainty.fasta.gz)
-   [ancient0004_lt120bp.fasta.gz](sra/lt120bp/consensus_sequence/ancient0004_lt120bp.fasta.gz)
-   [ancient0004_lt120bp.fasta_refmod.fasta.gz](sra/lt120bp/consensus_sequence/ancient0004_lt120bp.fasta_refmod.fasta.gz)
-   [ancient0004_lt120bp.fasta_uncertainty.fasta.gz](sra/lt120bp/consensus_sequence/ancient0004_lt120bp.fasta_uncertainty.fasta.gz)

### SRA, reads < lt130bp, rCRS

#### multiqc

-   [multiqc_report.html](sra/lt130bp/multiqc/multiqc_report.html)

##### multiqc_data

-   [mqc_ar_length_count_plot_All.txt](sra/lt130bp/multiqc/multiqc_data/mqc_ar_length_count_plot_All.txt)
-   [mqc_ar_length_count_plot_Collapsed.txt](sra/lt130bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed.txt)
-   [mqc_ar_length_count_plot_Collapsed_Truncated.txt](sra/lt130bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Collapsed_Truncated.txt)
-   [mqc_ar_length_count_plot_Discarded.txt](sra/lt130bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Discarded.txt)
-   [mqc_ar_length_count_plot_Mate1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate1.txt)
-   [mqc_ar_length_count_plot_Mate2.txt](sra/lt130bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Mate2.txt)
-   [mqc_ar_length_count_plot_Singleton.txt](sra/lt130bp/multiqc/multiqc_data/mqc_ar_length_count_plot_Singleton.txt)
-   [mqc_ar_retained_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_ar_retained_plot_1.txt)
-   [mqc_bcftools_stats_indel-lengths_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_bcftools_stats_indel-lengths_1.txt)
-   [mqc_fastqc_overrepresented_sequences_plot-2_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_overrepresented_sequences_plot-2_1.txt)
-   [mqc_fastqc_overrepresented_sequences_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_overrepresented_sequences_plot_1.txt)
-   [mqc_fastqc_per_base_n_content_plot-2_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot-2_1.txt)
-   [mqc_fastqc_per_base_n_content_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_base_n_content_plot_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot-2_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot-2_1.txt)
-   [mqc_fastqc_per_base_sequence_quality_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_base_sequence_quality_plot_1.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot-2_Percentages.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Counts.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Counts.txt)
-   [mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot-2_1.txt)
-   [mqc_fastqc_per_sequence_quality_scores_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_per_sequence_quality_scores_plot_1.txt)
-   [mqc_fastqc_sequence_counts_plot-2_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot-2_1.txt)
-   [mqc_fastqc_sequence_counts_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_sequence_counts_plot_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot-2_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot-2_1.txt)
-   [mqc_fastqc_sequence_duplication_levels_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_sequence_duplication_levels_plot_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot-2_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot-2_1.txt)
-   [mqc_fastqc_sequence_length_distribution_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fastqc_sequence_length_distribution_plot_1.txt)
-   [mqc_fiveprime_misinc_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_fiveprime_misinc_plot_1.txt)
-   [mqc_length-distribution-Forward_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_length-distribution-Forward_1.txt)
-   [mqc_length-distribution-Reverse_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_length-distribution-Reverse_1.txt)
-   [mqc_picard_deduplication_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_picard_deduplication_1.txt)
-   [mqc_preseq_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_preseq_plot_1.txt)
-   [mqc_qualimap_coverage_histogram_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_qualimap_coverage_histogram_1.txt)
-   [mqc_qualimap_gc_content_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_qualimap_gc_content_1.txt)
-   [mqc_qualimap_genome_fraction_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_qualimap_genome_fraction_1.txt)
-   [mqc_threeprime_misinc_plot_1.txt](sra/lt130bp/multiqc/multiqc_data/mqc_threeprime_misinc_plot_1.txt)
-   [multiqc.log](sra/lt130bp/multiqc/multiqc_data/multiqc.log)
-   [multiqc_adapter_removal.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_adapter_removal.txt)
-   [multiqc_bcftools_stats.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_bcftools_stats.txt)
-   [multiqc_citations.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_citations.txt)
-   [multiqc_damageprofiler_metrics.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_damageprofiler_metrics.txt)
-   [multiqc_data.json](sra/lt130bp/multiqc/multiqc_data/multiqc_data.json)
-   [multiqc_fastqc.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_fastqc.txt)
-   [multiqc_fastqc_1.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_fastqc_1.txt)
-   [multiqc_general_stats.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_general_stats.txt)
-   [multiqc_mtnucratio.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_mtnucratio.txt)
-   [multiqc_picard_dups.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_picard_dups.txt)
-   [multiqc_qualimap_bamqc_genome_results.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_qualimap_bamqc_genome_results.txt)
-   [multiqc_samtools_flagstat.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_samtools_flagstat.txt)
-   [multiqc_samtools_flagstat_1.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_samtools_flagstat_1.txt)
-   [multiqc_software_versions.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_software_versions.txt)
-   [multiqc_sources.txt](sra/lt130bp/multiqc/multiqc_data/multiqc_sources.txt)
-   [picard_histogram.txt](sra/lt130bp/multiqc/multiqc_data/picard_histogram.txt)
-   [picard_histogram_1.txt](sra/lt130bp/multiqc/multiqc_data/picard_histogram_1.txt)
-   [picard_histogram_2.txt](sra/lt130bp/multiqc/multiqc_data/picard_histogram_2.txt)
-   [preseq.txt](sra/lt130bp/multiqc/multiqc_data/preseq.txt)

#### consensus_sequence

-   [ancient0002_lt130bp.fasta.gz](sra/lt130bp/consensus_sequence/ancient0002_lt130bp.fasta.gz)
-   [ancient0002_lt130bp.fasta_refmod.fasta.gz](sra/lt130bp/consensus_sequence/ancient0002_lt130bp.fasta_refmod.fasta.gz)
-   [ancient0002_lt130bp.fasta_uncertainty.fasta.gz](sra/lt130bp/consensus_sequence/ancient0002_lt130bp.fasta_uncertainty.fasta.gz)
-   [ancient0003_lt130bp.fasta.gz](sra/lt130bp/consensus_sequence/ancient0003_lt130bp.fasta.gz)
-   [ancient0003_lt130bp.fasta_refmod.fasta.gz](sra/lt130bp/consensus_sequence/ancient0003_lt130bp.fasta_refmod.fasta.gz)
-   [ancient0003_lt130bp.fasta_uncertainty.fasta.gz](sra/lt130bp/consensus_sequence/ancient0003_lt130bp.fasta_uncertainty.fasta.gz)
-   [ancient0004_lt130bp.fasta.gz](sra/lt130bp/consensus_sequence/ancient0004_lt130bp.fasta.gz)
-   [ancient0004_lt130bp.fasta_refmod.fasta.gz](sra/lt130bp/consensus_sequence/ancient0004_lt130bp.fasta_refmod.fasta.gz)
-   [ancient0004_lt130bp.fasta_uncertainty.fasta.gz](sra/lt130bp/consensus_sequence/ancient0004_lt130bp.fasta_uncertainty.fasta.gz)
